sysutils/flash/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+
+github.setup    hypriot flash 2.3.0
+
+platforms       darwin
+categories      sysutils
+license         MIT
+maintainers     @jrjsmrtn openmaintainer
+description     Command line script to flash SD card images of any kind
+long_description    ${description}
+
+checksums       rmd160  d0f45b33da3b557cbf9253a835678310ad9ed27a \
+                sha256  7a31c2fa6a2169633c5d77b205ead4562b110e1c106b3ac4fa72ee55350e0719 \
+                size    16744
+
+depends_run     port:unzip
+
+variant curl description {If you want to flash directly with an HTTP URL} {
+    depends_run-append port:curl
+}
+
+variant aws description {If you want to flash directly from an AWS S3 bucket} {
+    depends_run-append port:py-awscli
```

Comment:
Any py-FOO port is just a stub and thus any python dependency needs to specify a version for consistency. Just use the latest stable version. In this case, change it to `py37-awscli`.

Reply:
Fixed.